aliyun/src/test/java/org/apache/iceberg/aliyun/oss/AliyunOSSTestRule.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aliyun.oss;
+
+import com.aliyun.oss.OSS;
+import java.util.UUID;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public interface AliyunOSSTestRule extends TestRule {
+  Logger LOG = LoggerFactory.getLogger(AliyunOSSTestRule.class);
+  UUID RANDOM_UUID = java.util.UUID.randomUUID();
+  /**
```

Comment:
In apache iceberg project, we usually leave an empty line before the two code blocks between two methods declaration.  I guess you need to check your intellij code style configuration and also apply the iceberg [checkstyles](https://iceberg.apache.org/community/#setting-up-ide-and-code-style) from the document. 

![image](https://user-images.githubusercontent.com/5028729/132298216-ab4cbdb3-43d1-4bad-bbe9-4e5d148be100.png)

Reply:
I have checked the idea style file "intellij-java-palantir-style.xml",  the default style between two method in the interface is empty as below 
![image](https://user-images.githubusercontent.com/12458911/133087831-216e217b-46a2-4915-b7c5-66e3468d6905.png)
let me know if the style file is incorrect

Reply:
I think it's worth to publish a PR to correct the option value in the [intellij-java-palantir-style.xml](https://github.com/apache/iceberg/blob/master/.baseline/idea/intellij-java-palantir-style.xml#L454).